sdk/storage/azure_storage_blob/tests/blob_client.rs

```diff
@@ -6,11 +6,13 @@ use azure_core::{
     Bytes,
 };
 use azure_core_test::{recorded, TestContext};
-use azure_storage_blob::models::BlobClientDownloadResultHeaders;
+use azure_storage_blob::clients::BlobClient;
 use azure_storage_blob::{
-    clients::{BlobClient, BlobContainerClient},
-    models::{BlobClientGetPropertiesResultHeaders, BlockListType, BlockLookupList, LeaseState},
-    BlobClientOptions, BlobContainerClientOptions,
+    models::{
+        BlobClientDownloadResultHeaders, BlobClientGetPropertiesResultHeaders, BlockListType,
```

Comment:
Just checking if this is expected behavior, that customers must explicitly import the `ResultHeaders` to be able to access the convenience methods i.e. `.content_length()`? If not, what is the appropriate way to refactor how this is currently imported and used in our code?